libraries/ms-drivers/src/spv1020_mppt.c

```diff
@@ -0,0 +1,25 @@
+#include "spv1020_mppt.h"
+#include "spv1020_mppt_defs.h"
+
+// This file implements common functionality between stm32 and x86. See stm32f0xx/spv1020_mppt.c
```

Comment:
Hmm, why is this file needed if it is neither for x86 or stm32?

Reply:
That file has common functionality, the `spv1020_is_X` functions are the same between x86 and stm32.

Reply:
Oh I see, so that saves you the from having to make the two files with duplicate functions